src/api/app/views/person/token/index.xml.builder

```diff
@@ -1,6 +1,7 @@
 xml.directory(count: @list.length) do |dir|
   @list.each do |token|
-    p = { id: token.id, string: token.string, kind: token.token_name, description: token.description, triggered_at: token.triggered_at }
+    token_name = token.token_name.gsub('service', 'runservice') # To make token naming consistent: we create the token as runservice
```

Comment:
```suggestion
    token_name = token.token_name.sub('service', 'runservice') # To make token naming consistent: we create the token as runservice
```
I would use `sub` instead of `gsub`. We don't expect to substitute more ocurrences.